editor/editor_node.cpp

```diff
@@ -5047,7 +5047,12 @@ void EditorNode::_save_docks_to_config(Ref<ConfigFile> p_layout, const String &p
 
 	// Save FileSystemDock state.
 
-	p_layout->set_value(p_section, "dock_filesystem_split", FileSystemDock::get_singleton()->get_split_offset());
+	if (unlikely(p_layout->has_section_key(p_section, "dock_filesystem_split"))) {
+		p_layout->erase_section_key(p_section, "dock_filesystem_split");
+	}
```

Comment:
I think there is no reason to remove this value. It will just stay unused if it exists.